Apps/Sandcastle/gallery/development/Ground Primitive.html

```diff
@@ -0,0 +1,152 @@
+    url: '//cesiumjs.org/stk-terrain/tilesets/world/tiles',
+    requestVertexNormals : true
+});
+
+// Circle geometry
+scene.primitives.add(new Cesium.GroundPrimitive({
+    geometryInstances: new Cesium.GeometryInstance({
+        geometry : new Cesium.CircleGeometry({
+            center : Cesium.Cartesian3.fromDegrees(-95.0, 45.0),
+            radius : 250000.0
+        }),
+        attributes : {
+            color : Cesium.ColorGeometryInstanceAttribute.fromColor(new Cesium.Color(1.0, 0.0, 0.0, 0.5))
+        }
+    })
+}));
+
+// Ellipse Geometry
+scene.primitives.add(new Cesium.GroundPrimitive({
+    geometryInstances: new Cesium.GeometryInstance({
+        geometry : new Cesium.EllipseGeometry({
+            center : Cesium.Cartesian3.fromDegrees(-105.0, 40.0),
+            semiMinorAxis : 300000.0,
+            semiMajorAxis : 400000.0
+        }),
+        attributes : {
+            color : Cesium.ColorGeometryInstanceAttribute.fromColor(new Cesium.Color(1.0, 1.0, 0.0, 0.5))
+        }
+    })
+}));
+
+// Corridor Geometry
+scene.primitives.add(new Cesium.GroundPrimitive({
+    geometryInstances : new Cesium.GeometryInstance({
+        geometry: new Cesium.CorridorGeometry({
+            positions : Cesium.Cartesian3.fromDegreesArray([
+                    -112.0, 40.0,
+                    -117.0, 40.0,
+                    -117.0, 35.0
+                ]),
+                width : 200000.0
+            }),
+        attributes : {
+            color : Cesium.ColorGeometryInstanceAttribute.fromColor(new Cesium.Color(0.0, 0.0, 1.0, 0.5))
+        }
+    })
+}));
+
+// Rectangle geometry
+scene.primitives.add(new Cesium.GroundPrimitive({
+    geometryInstances : new Cesium.GeometryInstance({
+        geometry : new Cesium.RectangleGeometry({
+            rectangle : Cesium.Rectangle.fromDegrees(-100.0, 30.0, -90.0, 40.0),
+            rotation : Cesium.Math.toRadians(45)
+        }),
+        attributes: {
+            color: Cesium.ColorGeometryInstanceAttribute.fromColor(new Cesium.Color(0.0, 1.0, 0.0, 0.5))
+        }
+    })
+}));
+
+// Polygon on Mount Saint Helens
+var positions = [new Cesium.Cartesian3(-2358138.847340281, -3744072.459541374, 4581158.5714175375),
+                 new Cesium.Cartesian3(-2357231.4925370603, -3745103.7886602185, 4580702.9757762635),
+                 new Cesium.Cartesian3(-2355912.902205431, -3744249.029778454, 4582402.154378103),
+                 new Cesium.Cartesian3(-2357208.0209552636, -3743553.4420488174, 4581961.863286629)];
+var polygonHierarchy = { positions : positions };
+
+var primitive = scene.primitives.add(new Cesium.GroundPrimitive({
+    geometryInstances : new Cesium.GeometryInstance({
+        geometry : new Cesium.PolygonGeometry({
+            polygonHierarchy : polygonHierarchy
+        }),
+        attributes: {
+            color: Cesium.ColorGeometryInstanceAttribute.fromColor(Cesium.Color.RED.withAlpha(0.5))
+        },
+        id : 'polygon1'
```

Comment:
Did we want to leave this `id`?

Or...how about we do mouseover picking in this example like we do for 3D models for testing?